### PR TITLE
Windows upload fix

### LINKED ISSUE #549: Unable to upload images
When I to go add new media, and click the "Upload media..." button and select a new image file I get some errors in the log and the media object isn't created:

```
2015-04-11T16:59:23.174Z - warn: [1] MediaService: Failed to select media renderer URI [/media/2015/4/a604b264-31ca-49b9-b880-6ca286bdc6b2-1428771563159.jpg]
2015-04-11T16:59:23.174Z - warn: [1] MediaService: No media renderer could be found for URI [/media/2015/4/a604b264-31ca-49b9-b880-6ca286bdc6b2-1428771563159.jpg]
2015-04-11T17:00:02.235Z - info: [1] Removed temporary file: C:\Users\Mark\AppData\Local\Temp\upload_03086ef0d5e2cd4e6e2cd1d7310cfb63
2015-04-11T17:00:02.248Z - warn: [1] MediaService: Failed to select media renderer URI [/media/2015/4/92644fd4-cbca-478c-be95-99c393d739b3-1428771602231.png]
2015-04-11T17:00:02.248Z - warn: [1] MediaService: No media renderer could be found for URI [/media/2015/4/92644fd4-cbca-478c-be95-99c393d739b3-1428771602231.png]
2015-04-11T17:04:41.926Z - info: [1] Removed temporary file: C:\Users\Mark\AppData\Local\Temp\upload_544581e8e05b55560769a121c3fa6222
2015-04-11T17:04:41.945Z - warn: [1] MediaService: Failed to select media renderer URI [/media/2015/4/e76f22ee-5a4f-4ada-8a6b-ffa4ddb3a47a-1428771881918.png]
2015-04-11T17:04:41.945Z - warn: [1] MediaService: No media renderer could be found for URI [/media/2015/4/e76f22ee-5a4f-4ada-8a6b-ffa4ddb3a47a-1428771881918.png]
```

When I import a blog via Wordpress or Blogger XML extracts, all those media objects are created just fine, and show up just fine on my site, but I can't create a new post and insert new media.

### PR DESCRIPTION
**Pull request by markbaird**
Fixes #549 - Makes the changes as discussed in issue #549 to fix media upload when running PencilBlue on a Windows machine.

### CODE REVIEW

File: include/util.js [hunk 1]
```diff
@@ -705,7 +705,8 @@ Util.getExtension = function(filePath, options) {
     
     //do to the end of the path
     var pathPartIndex = filePath.lastIndexOf(options.sep || path.sep) || 0;
-    filePath = filePath.substr(pathPartIndex);
+    if (pathPartIndex > -1)
+    	filePath = filePath.substr(pathPartIndex);
```
Comment (brianhyder) [inline include/util.js]: Would you mind including brackets with the if statement?  Coveralls yells at us over it.

**Comment by brianhyder** [comment]
Can you also include the same fix for the video_media_renderer and the audio_media_renderer?

Thank you again for diving into this one for us.

**Comment by markbaird** [comment]
Yeah, I'll work on that later this weekend.

**Comment by markbaird** [comment]
The requested changes are there now.

**Comment by brianhyder** [comment]
Thank you!

**Pull request merged**